conf/messages

```diff
@@ -1297,8 +1297,8 @@ tai.workingHours.legend=Your working hours are
 tai.workingHours.preHeading=Update your estimated income from {0}
 tai.workingHours.question=Are your working hours on each payslip generally the same or very different?
 tai.workingHours.regularHours=Hours are usually similar or the same from week to week
-tai.workingHours.heading=What are your working hours like through the year?
-tai.workingHours.title=What are your working hours like through the year?
+tai.workingHours.heading=What are your working hours through the year?
+tai.workingHours.title=What are your working hours through the year?
```

Comment:
We need the welsh translation for this

Reply:
Thanks @brendanstraughan  - check commen t on ticket re: Welsh 
https://jira.tools.tax.service.gov.uk/browse/PAYEE-103

Reply:
cool, will approve when I know a bit more about this force push

Reply:
Discussed this with Suchet and the Welsh translation is correct